variables.tf

```diff
@@ -279,7 +279,7 @@ variable "efs_volumes" {
     }))
   }))
 
-  description = "Task EFS volume definitions as list of configuration objects. You cannot define both Docker volumes and EFS volumes on the same task definition."
+  description = "Task EFS volume definitions as list of configuration objects. You can define only one of Docker volumes, EFS volumes and FSx volumes on the same task definition."
```

Comment:
@gauravkohli just wanted to clarify the var description: when you say "one of" does that mean "one of each type" or "one type"? It seems mixing types is supported (just based on the AWS docs)

Reply:
@joe-niland .. comment change was just following the old comment which looks like meant we can only have either Docker volumes or EFS volumes on the same task definition. So I added we can only have one type, but multiple volumes of the same type. 

But then I tested it on the ECS task and actually as you said we can indeed mix types, so we can have a task definition that can have all Docker, EFS & Fsx volumes attached to it and AWS doesn't complain about it.

Reply:
maybe useful info to add would be that the list `fsx_windows_file_server_volume_configuration` can only have one element, as that list is used for the dynamic `volume` block, and each `volume` block can only have one of `fsx_windows_file_server_volume_configuration`

Reply:
@gauravkohli that sounds like a good change, as well as perhaps changing 'only one of' to 'multiple'?